Aptoide/appcoins-wallet-android
# Feature/appc 595 get transaction from backend

**What does this PR do?**

   This PR changes the source of transactions shown at transactions screen from Trust api to our backend api

**Database changed?**

   Yes

**Where should the reviewer start?**

- [ ] FetchTransactionsInteract.java
- [ ] TransactionRepositoryType.java and its implementations

**How should this be manually tested?**

open the dev version of the wallet and import or generate transactions or payments. Those actions should appear at transactions screen.
 
**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-595](https://aptoide.atlassian.net/browse/APPC-595)


**Code Review Checklist**
